litmus-portal/frontend/src/components/YamlEditor/Editor.tsx

```diff
@@ -482,14 +368,14 @@ const YamlEditor: React.FC<YamlEditorProps> = ({
             editor.setAnimatedScroll(true);
             editor.setShowInvisibles(false);
             editor.setFontSize('0.98rem');
-            editor.container.style.background = palette.common.black;
+            // editor.container.style.background = palette.common.black;
             editor.container.style.lineHeight = '160%';
             const nodeStyle = (document.getElementsByClassName(
               'ace_gutter'
```

Comment:
Since we have updated the theme we should also update the gutter colours @S-ayanide

Reply:
Yes have updated the gutter color